locales/bn/bn.json

```diff
@@ -280,96 +280,96 @@
     "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "আপনি \"ক্লিক সমস্যা ভোগ করছি :action পাঠ্য\" বোতাম, কপি এবং নীচের ইউআরএল পেস্ট\nআপনার ওয়েব ব্রাউজারে:",
     "Increase": "বৃদ্ধি",
     "India": "ইন্ডিয়া",
-    "Indonesia": "ফিজি",
-    "Iran, Islamic Republic Of": "ইরান (ইসলামি প্রজাতন্ত্র)",
-    "Iran, Islamic Republic of": "ইরান (ইসলামি প্রজাতন্ত্র)",
-    "Iraq": "ফ্রান্স",
-    "Ireland": "ফরাসী পলিনেসিয়া",
+    "Indonesia": "ইন্দোনেশিয়া",
+    "Iran, Islamic Republic Of": "ইরান (ইসলামিক প্রজাতন্ত্র)",
+    "Iran, Islamic Republic of": "ইরান (ইসলামিক প্রজাতন্ত্র)",
+    "Iraq": "ইরাক",
+    "Ireland": "আয়ারল্যান্ড",
     "Isle Of Man": "আইল অব ম্যান",
-    "Isle of Man": "Isle of Man",
-    "Israel": "গ্যাবন",
+    "Isle of Man": "আইল অফ ম্যান",
+    "Israel": "ইজরাঈল",
     "It looks like you do not have an active subscription. You may choose one of the subscription plans below to get started. Subscription plans may be changed or cancelled at your convenience.": "দেখে মনে হচ্ছে আপনার একটি সক্রিয় সদস্যতা নেই৷ আপনি শুরু করতে নীচের সদস্যতা পরিকল্পনাগুলির মধ্যে একটি বেছে নিতে পারেন৷ সাবস্ক্রিপশন প্ল্যানগুলি আপনার সুবিধামত পরিবর্তন বা বাতিল হতে পারেন",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`).

This translation was by machine.